ui/app/templates/components/get-credentials-card.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="selectable-card is-rounded no-flex">
```

Comment:
well done. Should have been this in the first place, thank you.

Reply:
One of those things you don't catch until you're actually using it 😅